RadixWallet/Features/AssetTransferFeature/AssetTransfer+Reducer.swift

```diff
@@ -258,8 +258,8 @@ func instructionForDepositing(
 
 	return try ManifestBuilder.accountTryDepositOrAbort(
 		recipientAddress.intoEngine(),
-		nil,
-		bucket
+		bucket,
+		nil
```

Comment:
@GhenadieVP is the flip of arguments here intended? A change in RET?

Reply:
yes, the order flipped.